### PR TITLE
Bug 558830 LIC Define feature must have it's own branding plug-in

### PR DESCRIPTION
**Pull request by eparovyshnaya**
`org.eclipse.passage.lic.define.branding` bundle contains branding for
the LIC Define feature

Signed-off-by: elena.parovyshnaya <elena.parovyshnaya@gmail.com>

### CODE REVIEW

**Comment by ruspl-afed** [comment]
Check aggregator pom

File: bundles/org.eclipse.passage.lic.define.branding/OSGI-INF/l10n/bundle.properties [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     ArSysOp - initial API and implementation
+###############################################################################
+
+Bundle-Name = Passage LIC Define Branding
+Bundle-Vendor = Eclipse Passage
+Bundle-Copyright = Copyright (c) 2020 ArSysOp and others.\n\
```
[the remark this answers is not in the document]
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.define.branding/OSGI-INF/l10n/bundle.properties]: lic.define.branding is a brand-new plug-in created in 2020

File: bundles/org.eclipse.passage.lic.define.branding/.classpath [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
+	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
+	<classpathentry kind="output" path="target/classes/"/>
```
[the remark this answers is not in the document]
Comment (eparovyshnaya) [reply bundles/org.eclipse.passage.lic.define.branding/.classpath]: done

**Comment by eparovyshnaya** [comment]
Resolves [558830](https://bugs.eclipse.org/bugs/show_bug.cgi?id=558830)

**Pull request merged**